.travis.yml

```diff
@@ -16,8 +16,10 @@ matrix:
             - sourceline: 'ppa:cs50/ppa'
             - sourceline: 'ppa:ubuntu-toolchain-r/test'
             - sourceline: 'ppa:beineri/opt-qt-5.12.2-xenial'
+            - sourceline: 'deb http://apt.llvm.org/stretch/ llvm-toolchain-stretch-9 main'
+              key_url: 'https://apt.llvm.org/llvm-snapshot.gpg.key'
           packages:
-            - astyle
+            - clang-format-9
```

Comment:
Why do we need to uninstall and use clang-9?

It doesn't look like you are using any new formatting options.

Reply:
No new options, but clang-format 7 does have some bugs for this features.